tests/acceptance/features/webUIPreview/imageMediaViewer.feature

```diff
@@ -78,3 +78,15 @@ Feature: display image in media viewer on the webUI
     And user "user1" has logged in using the webUI
     When the user views the file "testimage.mp3" in the media viewer using the webUI
     Then the file "testimage.mp3" should be displayed in the media viewer webUI
+
+  Scenario: preview of png file with media viewer is possible
+    Given user "user1" has uploaded file "testavatar.png" to "testavatar.png"
+    And user "user1" has logged in using the webUI
+    When the user views the file "testavatar.png" in the media viewer using the webUI
+    Then the file "testavatar.png" should be displayed in the media viewer webUI
+
+  Scenario: preview of jpeg with media viewer is possible
+    Given user "user1" has uploaded file "testavatar.jpeg" to "testavatar.jpeg"
+    And user "user1" has logged in using the webUI
+    When the user views the file "testavatar.jpeg" in the media viewer using the webUI
+    Then the file "testavatar.jpeg" should be displayed in the media viewer webUI
```

Comment:
You can use data table in the first scenario. @swoichha  please assist her

Reply:
Also please remove `tests/acceptance/filesForUpload/testavatar.jpeg` from git

Reply:
I think including both of these scenarios in a single scenario outline will be nice.